src/data/right_data.py

```diff
@@ -17,3 +17,70 @@ def __init__(self, *args, **kwargs):
         super().__init__(*args, **kwargs)
 
         self.dataset = RightDaVinciDataSet
+
```

Comment:
why did you need to add this?

Reply:
it looks pretty much the same as setup() no?

Reply:
might be worth just writing right data not as subclass?

Reply:
it's setup without the shuffle in the middle
removed:
```
        # shuffle all 41 sets of 1000 frames
        self.all_sets = shuffle(all_sets, random_state=42)
```